From b0884beee47d7c109e221b766eb44d3153545ae7 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Fri, 1 Jun 2018 14:04:51 +0800
Subject: [PATCH] ASoC: rk3308_codec: update properties

Remove 'rockchip,adc-grps' and add 'rockchip,enable-all-adcs'

Change-Id: Ic838df1b26fa4d5837368025b5c335b8cad5226c
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 .../bindings/sound/rockchip,rk3308-codec.txt           | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/Documentation/devicetree/bindings/sound/rockchip,rk3308-codec.txt b/Documentation/devicetree/bindings/sound/rockchip,rk3308-codec.txt
index 6889483cec49..b7345d29f5dd 100644
--- a/Documentation/devicetree/bindings/sound/rockchip,rk3308-codec.txt
+++ b/Documentation/devicetree/bindings/sound/rockchip,rk3308-codec.txt
@@ -13,17 +13,15 @@ Required properties:
 - reset-names : Must include the following entries: "acodec-reset".
 
 Optional properties:
-- rockchip,adc-grps: This is a variable length array, that shows the mapping of
-  the configuration adc grp. The following shows the relationship between grp
+- rockchip,enable-all-adcs: This is a boolean type property, that shows whether
+  force enable all of ADCs. The following shows the relationship between grps
   and ADC:
   * grp 0 -- select ADC1 / ADC2
   * grp 1 -- select ADC3 / ADC4
   * grp 2 -- select ADC5 / ADC6
   * grp 3 -- select ADC7 / ADC8
-  By default, they are all enabled, like using:
-  - rockchip,adc-grps = <0 1 2 3>;
-  If we don't use ADC5 and ADC6, we just use grp 0,1,3 on dts. For example:
-  - rockchip,adc-grps = <0 1 3>;
+  If the property is not used, the enabled ADC groups refer to needed channels
+  via configure hw_params.
 
 - rockchip,adc-grps-route: This is a variable length array, that shows the
   mapping route of ACODEC sdo to I2S sdi. By default, they are one-to-one
-- 
2.35.3

